src/clients/jellyfin/jellyfin.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { Api, Jellyfin } from '@jellyfin/sdk';
+import { Constants } from '../../utils/constants';
+import { SystemApi } from '@jellyfin/sdk/lib/generated-client/api/system-api';
+import { getSystemApi } from '@jellyfin/sdk/lib/utils/api/system-api';
+
+@Injectable()
+export class JellyfinService {
+  private readonly logger = new Logger(JellyfinService.name);
+  private jellyfin: Jellyfin;
+  private api: Api;
+  private systemApi: SystemApi;
+  private userId: string;
+
+  init() {
+    this.jellyfin = new Jellyfin({
+      clientInfo: {
+        name: Constants.Metadata.ApplicationName,
+        version: Constants.Metadata.Version,
+      },
+      deviceInfo: {
+        id: 'test',
```

Comment:
change to something more specific
Maybe
id: random uuid
name: Jellyfin Discord Bot